src/outputs/short_form_out.py

```diff
@@ -23,20 +23,22 @@ def create_period_year(df: pd.DataFrame) -> pd.DataFrame:
 
 
 def create_headcount_cols(
-    df: pd.DataFrame,
-    round_val: int = 4,
+    df: pd.DataFrame, fte_civil, fte_defence, hc_total, round_val=4
 ) -> pd.DataFrame:
     """Create new columns with headcounts for civil and defence.
 
-    Column '705' contains the total headcount value, and
+    The column represented by hc_total contains the total headcount value, and
     from this the headcount values for civil and defence are calculated
-    based on the percentages of civil and defence in columns '706' (civil)
-    and '707' (defence). Note that columns '706' and '707' measure different
-    things to '705' so will not in general total to the '705' value.
+    based on the percentages of civil and defence in fte_civil (civil)
+    fte_defence (defence). Note that fte_civil and fte_defence measure different
+    things to hc_total so will not in general total to the hc_total value.
 
     Args:
         df (pd.DataFrame): The survey dataframe being prepared for
             short form output.
+        fte_civil (str): Column containing percentage of civil employees.
+        fte_defence (str): Column containing percentage of defence employees.
+        hc_total (str): Column containing total headcount value.
```

Comment:
It would be nice to add that fte_civil is usually 706, etc

Reply:
Docstring updated